src/app/public/modules/text-editor/services/text-editor-adapter.service.ts

```diff
@@ -213,25 +207,38 @@ export class SkyTextEditorAdapterService {
   public focusEditor(id: string): void {
     if (id in this.editors) {
       const windowEl = this.getContentWindowEl(id);
-      const documentEl = this.windowService.nativeWindow.document;
-
-      const editor: any = this.getDocumentEl(id).body;
-      const range = documentEl.createRange();
-
-      this.editors[id].iframeElementRef.focus();
-      editor.focus();
-      /* istanbul ignore else */
-      if (windowEl.getSelection && documentEl.createRange) {
-        range.selectNodeContents(editor);
-        range.collapse(false);
-        const sel = windowEl.getSelection();
-        sel.removeAllRanges();
-        sel.addRange(range);
+      const iframeDocumentEl = this.getDocumentEl(id);
```

Comment:
I was initially confused by the variable name `iframeDocumentEl` when the method call is `getDocumentEl`. Should the method be renamed to `getIframeDocumentEl` since you can get the document both inside and outside of an iframe?

Reply:
That's fair. Updated.